tests/cases/fourslash/jsxExpressionFollowedByIdentifier.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path="fourslash.ts" />
+
+//@Filename: jsxExpressionFollowedByIdentifier.tsx
+////declare var React: any;
+////const a = <div>{<div />[|x|]}</div>
+////const b = <div x={<div />[|x|]} />
+
+test.ranges().forEach(range => {
+  verify.errorExistsAtRange(range, 1005, "'}' expected.");
```

Comment:
Maybe `ts.Diagnostics.<message>.code` (or equivalent reexposed to the test environment) would be better, so we avoid hardcoding an error code into the test.

Reply:
Humm, this would be easy enough if the generated diagnostics were a module, but it’s currently an `@internal` slice of the `ts` namespace. Changing/ammending how those get generated might be beyond what I want to cover in the scope of this PR. Writing it down to revisit later. FWIW, I don't like it, but all the existing error stuff in fourslash uses hardcoded messages and codes 😬 Maybe I'll come back and give all those methods a little TLC.

Reply:
A triple-slash ref to `diagnosticInformationMap.generated.ts` in the `tests/cases/fourslash/fourslash.ts` file aughta do it, but I dunno how that'll affect performance of the foruslash tests.

Reply:
Whaaaaat you can triple slash `@internal` things?! 😮

Reply:
Well, yeah - the file's not internal or anything, just the namespace inside it. The `@internal` annotation only affects declaration emit.

Reply:
Before:
```
real    1m13.848s
user    9m52.079s
sys     0m23.487s
```

After:
```
real    1m13.816s
user    9m55.116s
sys     0m23.758s
```

Think it's worth the performance hit? 😛

Reply:
oh, well that's like nothing. yeah, seems fine.